tests/attr/test_gradient_shap.py

```diff
@@ -46,7 +49,7 @@ def test_basic_multi_input(self):
         ig = IntegratedGradients(model)
         baselines = (torch.zeros(batch_size, 3), torch.zeros(batch_size, 4))
         attributions_ig = ig.attribute(inputs, baselines=baselines)
-        self._assert_shap_ig_comparision(attributions, attributions_ig)
+        self._assert_shap_ig_comparision(tuple(attributions), attributions_ig)
```

Comment:
Instead of using tuple here, which isn't necessary at runtime, can you instead change line 42 above to:
```
        attributions, delta = cast(
            Tuple[Tuple[Tensor, ...], Tensor],
            gradient_shap.attribute(
                inputs, baselines, n_samples=n_samples, return_convergence_delta=True
            ),
        )

```
Using cast (imported from typing) has no effect at runtime and will just make the type explicit to the type checker.